packages/commonwealth/client/scripts/views/pages/Community/Groups/TopicGatingHelpMessage/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CWText } from 'views/components/component_kit/cw_text';
+import './index.scss';
+
+const TopicGatingHelpMessage = () => {
+  return (
+    <section className="TopicGatingHelpMessage">
```

Comment:
generally I think this might be a new CWBanner (alert) component but maybe design team had different desired. Imo we should be reusing components from the design system

Reply:
> Imo we should be reusing components from the design system

Hm, I couldn't find a similar one. Do you know if we already have a similar one?

Reply:
As I said, [CWBanner (alert)](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=793%3A11228&mode=dev). It is not identical so this would need require feedback from design, if we should reuse alert component or create custom one as you did